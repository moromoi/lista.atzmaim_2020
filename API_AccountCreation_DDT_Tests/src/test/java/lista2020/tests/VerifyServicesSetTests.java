package lista2020.tests;

import org.approvaltests.Approvals;
import org.testng.annotations.Test;

import java.util.HashSet;

import static org.testng.Assert.assertEquals;

public class VerifyServicesSetTests extends TestBase {

    @Test(priority = 1)
    public void getHolisticJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[1]"));

    }

    @Test(priority = 2)
    public void getConsultansJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[2]"));
    }

    @Test(priority = 3)
    public void getCosmeticsJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[3]"));
    }

    @Test(priority = 4)
    public void getHairSalonJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[4]"));
    }

    @Test(priority = 5)
    public void getInteriorJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[5]"));
    }

    @Test(priority = 6)
    public void getInstallersJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[6]"));
    }

    @Test(priority = 7)
    public void getMakeupJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[7]"));
    }

    @Test(priority = 8)
    public void getNailJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[8]"));
    }

    @Test(priority = 9)
    public void getMassageJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[9]"));
    }

    @Test(priority = 10)
    public void getMoversJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[10]"));
    }

    @Test(priority = 11)
    public void getNutritionistsJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[11]"));
    }

    @Test(priority = 12)
    public void getPestJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[12]"));
    }

    @Test(priority = 13)
    public void getTeachersJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[13]"));
    }

    @Test(priority = 14)
    public void getSalesJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[14]"));
    }

    @Test(priority = 15)
    public void getServiceProvidersJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[15]"));
    }

    @Test(priority = 16)
    public void getPersonalJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[16]"));
    }

    @Test(priority = 17)
    public void getTattooJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[17]"));
    }

    @Test(priority = 18)
    public void getTherapistsJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[18]"));
    }

    @Test(priority = 19)
    public void getAllTypeJSON() {
        for (int i = 0; i < 1; i++) {

            Approvals.verify(app.serviceRestHelper().createAccountWithType("[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18]"));
        }
    }

    @Test(priority = 20)
    public void withoutTypeJSON() {
        for (int i = 0; i < 1; i++) {
            Approvals.verify(app.serviceRestHelper().createAccountWithType("[]"));
        }
    }




    @Test(priority = 21)
    public void deleteAccount() {

        assertEquals(app.serviceRestHelper().deleteAccount(), new HashSet<>() );
    }


}