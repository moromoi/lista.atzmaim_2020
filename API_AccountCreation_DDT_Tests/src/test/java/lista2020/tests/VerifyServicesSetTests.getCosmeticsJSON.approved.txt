Server=nginx
###Date##
Content-Type=application/json; charset=utf-8
Transfer-Encoding=chunked
Connection=keep-alive
Vary=Accept-Encoding
###Set-Cookie##
###X-Request-Id##
Strict-Transport-Security=max-age=604800
X-Content-Type-Options=nosniff
X-Robots-Tag=noindex
Content-Security-Policy=upgrade-insecure-requests
X-Powered-By=php/4.3.10-16
Content-Encoding=gzip

[
    {
        "id": 1,
        "name": "Tipul panim",
        "price": "350",
        "duration": 60,
        "color": "#50e3c1",
        "category": {
            "name": "Skin Treatments",
            "id": 1
        }
    },
    {
        "id": 2,
        "name": "Tipul yoffi/zoer",
        "price": "250",
        "duration": 45,
        "color": "#50e3c1",
        "category": {
            "name": "Skin Treatments",
            "id": 1
        }
    },
    {
        "id": 3,
        "name": "Tipul mezoterapya",
        "price": "400",
        "duration": 90,
        "color": "#50e3c1",
        "category": {
            "name": "Skin Treatments",
            "id": 1
        }
    },
    {
        "id": 4,
        "name": "Tipul ahdarat lahut",
        "price": "200",
        "duration": 60,
        "color": "#50e3c1",
        "category": {
            "name": "Skin Treatments",
            "id": 1
        }
    },
    {
        "id": 5,
        "name": "Tipul pigmentasya",
        "price": "300",
        "duration": 75,
        "color": "#50e3c1",
        "category": {
            "name": "Skin Treatments",
            "id": 1
        }
    },
    {
        "id": 6,
        "name": "Tipul akne",
        "price": "250",
        "duration": 90,
        "color": "#50e3c1",
        "category": {
            "name": "Skin Treatments",
            "id": 1
        }
    },
    {
        "id": 7,
        "name": "Tipul antiaging / Mitzuk",
        "price": "300",
        "duration": 60,
        "color": "#50e3c1",
        "category": {
            "name": "Skin Treatments",
            "id": 1
        }
    },
    {
        "id": 8,
        "name": "Piling",
        "price": "400",
        "duration": 75,
        "color": "#50e3c1",
        "category": {
            "name": "Skin Treatments",
            "id": 1
        }
    },
    {
        "id": 9,
        "name": "Lak dgel",
        "price": "120",
        "duration": 45,
        "color": "#efefff",
        "category": {
            "name": "Manicure Pedicure",
            "id": 2
        }
    },
    {
        "id": 10,
        "name": "Lak dgel reglaim",
        "price": "150",
        "duration": 30,
        "color": "#efefff",
        "category": {
            "name": "Manicure Pedicure",
            "id": 2
        }
    },
    {
        "id": 11,
        "name": "Manikur express",
        "price": "45",
        "duration": 15,
        "color": "#efefff",
        "category": {
            "name": "Manicure Pedicure",
            "id": 2
        }
    },
    {
        "id": 12,
        "name": "Manikur",
        "price": "70",
        "duration": 30,
        "color": "#efefff",
        "category": {
            "name": "Manicure Pedicure",
            "id": 2
        }
    },
    {
        "id": 13,
        "name": "Pedikur express",
        "price": "55",
        "duration": 15,
        "color": "#efefff",
        "category": {
            "name": "Manicure Pedicure",
            "id": 2
        }
    },
    {
        "id": 14,
        "name": "Pedikur",
        "price": "140",
        "duration": 45,
        "color": "#efefff",
        "category": {
            "name": "Manicure Pedicure",
            "id": 2
        }
    },
    {
        "id": 15,
        "name": "Pedikur refui",
        "price": "155",
        "duration": 45,
        "color": "#efefff",
        "category": {
            "name": "Manicure Pedicure",
            "id": 2
        }
    },
    {
        "id": 16,
        "name": "Bnya + Lak dgel",
        "price": "240",
        "duration": 60,
        "color": "#759bfa",
        "category": {
            "name": "Nail Sculpting",
            "id": 3
        }
    },
    {
        "id": 17,
        "name": "Miluy + Lak dgel",
        "price": "130",
        "duration": 60,
        "color": "#759bfa",
        "category": {
            "name": "Nail Sculpting",
            "id": 3
        }
    },
    {
        "id": 18,
        "name": "Bnyat Bniyat tzipornaim",
        "price": "300",
        "duration": 60,
        "color": "#759bfa",
        "category": {
            "name": "Nail Sculpting",
            "id": 3
        }
    },
    {
        "id": 19,
        "name": "Bnya ?dgel",
        "price": "220",
        "duration": 60,
        "color": "#759bfa",
        "category": {
            "name": "Nail Sculpting",
            "id": 3
        }
    },
    {
        "id": 20,
        "name": "Tikun tzipornayim",
        "price": "0",
        "duration": 15,
        "color": "#759bfa",
        "category": {
            "name": "Nail Sculpting",
            "id": 3
        }
    },
    {
        "id": 21,
        "name": "Miluy ragil",
        "price": "110",
        "duration": 60,
        "color": "#759bfa",
        "category": {
            "name": "Nail Sculpting",
            "id": 3
        }
    },
    {
        "id": 22,
        "name": "Asarat Bnya/Lak dgel",
        "price": "50",
        "duration": 30,
        "color": "#759bfa",
        "category": {
            "name": "Nail Sculpting",
            "id": 3
        }
    },
    {
        "id": 23,
        "name": "Adbakat Bniyat tzipornaim",
        "price": "0",
        "duration": 30,
        "color": "#759bfa",
        "category": {
            "name": "Nail Sculpting",
            "id": 3
        }
    },
    {
        "id": 24,
        "name": "Sidur gabot",
        "price": "30",
        "duration": 15,
        "color": "#b8e986",
        "category": {
            "name": "Eyebrows / Mustache",
            "id": 4
        }
    },
    {
        "id": 25,
        "name": "Sapham",
        "price": "30",
        "duration": 15,
        "color": "#b8e986",
        "category": {
            "name": "Eyebrows / Mustache",
            "id": 4
        }
    },
    {
        "id": 26,
        "name": "Sapham + gabot",
        "price": "50",
        "duration": 30,
        "color": "#b8e986",
        "category": {
            "name": "Eyebrows / Mustache",
            "id": 4
        }
    },
    {
        "id": 27,
        "name": "Iuts Leizer",
        "price": "0",
        "duration": 30,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 28,
        "name": "Leizer yadaim",
        "price": "0",
        "duration": 30,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 29,
        "name": "Leizer reglaim",
        "price": "0",
        "duration": 45,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 30,
        "name": "Leizer bikini+shehi",
        "price": "0",
        "duration": 30,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 31,
        "name": "Leizer bikini",
        "price": "0",
        "duration": 30,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 32,
        "name": "Leizer shehi",
        "price": "0",
        "duration": 15,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 33,
        "name": "Leizer ?.regel ilyona",
        "price": "0",
        "duration": 30,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 34,
        "name": "Leizer ?.regel tahtona",
        "price": "0",
        "duration": 30,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 35,
        "name": "Leizer yashvan",
        "price": "0",
        "duration": 30,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 36,
        "name": "Leizer gav",
        "price": "0",
        "duration": 30,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 37,
        "name": "Leizer beten",
        "price": "0",
        "duration": 30,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 38,
        "name": "Leizer haze+beten",
        "price": "0",
        "duration": 45,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 39,
        "name": "Leizer ??r?",
        "price": "0",
        "duration": 15,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 40,
        "name": "Leizer panim",
        "price": "0",
        "duration": 15,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 41,
        "name": "Leizer Sapham+???r",
        "price": "0",
        "duration": 15,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 42,
        "name": "Leizer Sapham",
        "price": "0",
        "duration": 15,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 43,
        "name": "Leizer lahaim gavr",
        "price": "0",
        "duration": 15,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 44,
        "name": "Leizer haze",
        "price": "0",
        "duration": 30,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 45,
        "name": "Leizer pas beten",
        "price": "0",
        "duration": 15,
        "color": "#3fb6dc",
        "category": {
            "name": "Epilation",
            "id": 5
        }
    },
    {
        "id": 46,
        "name": "Shaava - panim",
        "price": "50",
        "duration": 15,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 47,
        "name": "Shaava - shehi",
        "price": "30",
        "duration": 15,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 48,
        "name": "Shaava - ??????",
        "price": "40",
        "duration": 15,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 49,
        "name": "Shaava - mafshaut",
        "price": "60",
        "duration": 15,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 50,
        "name": "Shaava - peot",
        "price": "80",
        "duration": 15,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 51,
        "name": "Shaava - reglaim",
        "price": "120",
        "duration": 30,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 52,
        "name": "Shaava - hatzi regel",
        "price": "80",
        "duration": 30,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 53,
        "name": "Shaava - reglaim + bikini",
        "price": "150",
        "duration": 30,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 54,
        "name": "Shaava - yadaim",
        "price": "80",
        "duration": 15,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 55,
        "name": "Shaava gavr - gav",
        "price": "120",
        "duration": 30,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 56,
        "name": "Shaava gavr - haze",
        "price": "80",
        "duration": 30,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 57,
        "name": "Shaava gavr - ??Fen??",
        "price": "50",
        "duration": 15,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 58,
        "name": "Shaava gavr - ??r?",
        "price": "30",
        "duration": 15,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 59,
        "name": "Shaava - beten",
        "price": "50",
        "duration": 15,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 60,
        "name": "Shaava gavr - beten",
        "price": "70",
        "duration": 15,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 61,
        "name": "Shaava - hatzi ??",
        "price": "40",
        "duration": 15,
        "color": "#deceff",
        "category": {
            "name": "Waxing",
            "id": 6
        }
    },
    {
        "id": 62,
        "name": "????r kavua- ???????r ?????",
        "price": "0",
        "duration": 120,
        "color": "#ff98ad",
        "category": {
            "name": "Makeup / Permanent Makeup",
            "id": 7
        }
    },
    {
        "id": 63,
        "name": "????r kavua - ???????r tahton",
        "price": "0",
        "duration": 120,
        "color": "#ff98ad",
        "category": {
            "name": "Makeup / Permanent Makeup",
            "id": 7
        }
    },
    {
        "id": 64,
        "name": "????r kavua - ??????",
        "price": "0",
        "duration": 120,
        "color": "#ff98ad",
        "category": {
            "name": "Makeup / Permanent Makeup",
            "id": 7
        }
    },
    {
        "id": 65,
        "name": "????? ????r kavua",
        "price": "0",
        "duration": 60,
        "color": "#ff98ad",
        "category": {
            "name": "Makeup / Permanent Makeup",
            "id": 7
        }
    },
    {
        "id": 66,
        "name": "????r ?r?",
        "price": "300",
        "duration": 45,
        "color": "#ff98ad",
        "category": {
            "name": "Makeup / Permanent Makeup",
            "id": 7
        }
    },
    {
        "id": 67,
        "name": "????r kala",
        "price": "950",
        "duration": 90,
        "color": "#ff98ad",
        "category": {
            "name": "Makeup / Permanent Makeup",
            "id": 7
        }
    },
    {
        "id": 68,
        "name": "????r kavua gabot",
        "price": "0",
        "duration": 120,
        "color": "#ff98ad",
        "category": {
            "name": "Makeup / Permanent Makeup",
            "id": 7
        }
    }
]